CommandsAndAdaptors/parser_adaptor.py

```diff
@@ -0,0 +1,13 @@
+class ParsedInput:
+    def __init__(self, args):
+        self.command = str(args.command).lower()
+        self.schema = args.schema
+        self.database = args.database
+        self.table = args.table
+        self.primary_key = args.primary_key
+        self.parameter = args.parameter
+        self.value = args.value
+
+    def parse(self):
+        a = [self.command, self.schema, self.database, self.table, self.primary_key, self.parameter, self.value]
```

Comment:
Why do you need this array and this method ?
You already have the variable here.

You didn’t understand the Adapter design pattern. I recommend read about it.